HW5/task1.py

```diff
@@ -0,0 +1,14 @@
+# Напишите функцию, которая принимает на вход строку - абсолютный путь до файла.
+# Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла.
+
+def input_text(text):
+    array_text = text.split("\\")
+    name_file = array_text[-1].split('.')
+    result = {text, name_file[0], name_file[1]}
+    return tuple(result)
```

Comment:
Да, можно проще. Я предложу свой код, но вы обязательно в нем разберитесь
```suggestion
def input_text(text):
    *path, file = text.rsplit(symbol, 1)
    return *path, *file.split('.')
```